src/TicketService.java

```diff
@@ -22,7 +24,7 @@ public static void main(String[] args) {
             Ticket ticketById = findById(searchId, tickets);
             System.out.println("Found ticket: " + ticketById);
         } catch (NoSuchElementException e) {
-            System.out.println("Ticket with id " + searchId + " not found.");
+            System.out.println("model.Ticket with id " + searchId + " not found.");
```

Comment:
**model.** why we need it?

Reply:
Hi! Sorry! It was automatic after I moved my classes to the new model folder. Thank you a lot ! I didn't notice.